tests/conftest.py

```diff
@@ -500,7 +500,7 @@ def setup_corrupted_role(request, ansible_module):
     role_name = "test_role"
     resource_type = gen_string("alpha")
     ansible_module.command(f"hammer role create --name {role_name}")
-    ansible_module.command(f"hammer filter create --role {role_name} --permission-ids 62,68")
+    ansible_module.command(f"hammer filter create --role {role_name} --permission-ids 51,57")
```

Comment:
@shubhamsg199 Update this use permission names instead, so we need to take care of ids being changed